maitake/src/task.rs

```diff
@@ -329,7 +363,7 @@ where
             "Task::poll"
         );
         let mut this = ptr.0.cast::<Self>();
-        test_trace!(task = ?fmt::alt(this.as_ref()));
+        test_debug!(task = ?fmt::alt(this.as_ref()));
```

Comment:
Did you want to keep this elevated?

Reply:
(there are a bunch more of these later in the PR too)

Reply:
these used to be logged at `DEBUG` previously, as well, the macro was (confusingly) named `test_trace!` but emitted a `DEBUG` event. the reason a bunch of them changed is because i renamed the macro and added a new `test_trace!` that emits a `TRACE` event.